### PR TITLE
Use mustache for constants + setup for polygon and quickswap pricing

### PR DESCRIPTION
**Pull request by IainM22**
Like we suspected, quickswap is a direct copy of uniswap so our uniswap pricing method ports over just fine. I ran an experiment to confirm, and pricing was working for WETH and a couple random ERC20s from quickswap. 

The meat of this PR is around utilizing mustache to generate `constants.ts` in order to better support multi-blockchain.

### CODE REVIEW

File: config/polygon.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "network": "matic",
+  "geyser_factory_v1": "0x0000000000000000000000000000000000000000",
+  "geyser_factory_v1_start_block": 16009661,
+  "pool_factory": "0x0000000000000000000000000000000000000000",
+  "pool_factory_start_block": 16009661,
+  "gysr_address": "0xbea98c05eeae2f3bc8c3565db7551eb738c8ccab",
```
Comment (devinaconley) [inline config/polygon.json]: should this be the mapped gysr token?
```
0xc48F61a288A08F1B80c2edd74652e1276B6A168c
```

File: config/polygon.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "link_address": "0x53e0bca35ec356bd5dddfebbd1fc0fd03fabad39",
+  "bnb_address": "0x0000000000000000000000000000000000000000",
+  "snx_address": "0x0000000000000000000000000000000000000000",
+  "yfi_address": "0x0000000000000000000000000000000000000000",
+  "comp_address": "0x0000000000000000000000000000000000000000",
+  "mkr_address": "0x0000000000000000000000000000000000000000",
+  "grt_address": "0x0000000000000000000000000000000000000000",
+  "usdt_weth_address": "0xf6422b997c7f54d1c6a6e103bcb1499eea0a7046",
+  "uniswap_factory": "0x5757371414417b8C6CAad45bAeF941aBc7d3Ab32",
+  "sushi_factory": "0x5757371414417b8C6CAad45bAeF941aBc7d3Ab32",
```
Comment (devinaconley) [inline config/polygon.json]: sushi is actually on polygon so we can use that address:
```
0xc35DADB65012eC5796536bD9864eD8773aBc74C4
```

File: config/kovan.json [hunk 3]
```diff
@@ -3,5 +3,24 @@
...
+  "snx_address": "0x0000000000000000000000000000000000000000",
+  "yfi_address": "0x0000000000000000000000000000000000000000",
+  "comp_address": "0x0000000000000000000000000000000000000000",
+  "mkr_address": "0x0000000000000000000000000000000000000000",
+  "grt_address": "0x0000000000000000000000000000000000000000",
+  "usdt_weth_address": "0x0000000000000000000000000000000000000000",
+  "uniswap_factory": "0x0000000000000000000000000000000000000000",
+  "sushi_factory": "0x0000000000000000000000000000000000000000",
+  "erc20_competitive_reward_module_factory_address": "0x5c1A02D6CAa8C954Cb30CE2B1a6C02Edd75fc3E3",
+  "erc20_friendly_reward_module_factory_address": "0x88fb2e8951534200411a4D6eBd7d000c225B76F0"
```
Comment (devinaconley) [inline config/kovan.json:25]: are these the right ones? only see a couple transactions on these factories

**Review by devinaconley — CHANGES_REQUESTED** [review]
looks great! just a couple minor ones

File: config/polygon.json (hunk 2 above)
Comment (IainM22) [reply config/polygon.json]: Nice, good call

File: config/polygon.json (hunk 1 above)
Comment (IainM22) [reply config/polygon.json]: Why yes it should. Good catch

File: config/kovan.json (hunk 3 above)
Comment (IainM22) [reply config/kovan.json:25]: Yeah, double checked and these are correct

**Review by devinaconley — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**